spec.md

```diff
@@ -491,6 +503,18 @@ The foundation block simply sequences operations and other blocks
 }
 ```
 
+### Parallel block
+
+A grouping of quantum operations to be performed in parallel
```

Comment:
See comments in the `model.py`. If this block is for quantum ops only, the name should reflect that.